src/arm/bbb-bone-buses.dtsi

```diff
@@ -370,59 +370,41 @@
 
 // UARTs
 bone_uart_1: &uart1 {
-	status = "disabled";
-	pinctrl-names = "default";
```

Comment:
I removed these because they serve no purpose as the settings are already done in the base files.